tests/NovelRT.Tests/Timing/TimestampTest.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT License (MIT). See LICENCE.md in the repository root for more information.
+
+#include <gtest/gtest.h>
+#include <NovelRT.h>
+
+using namespace NovelRT;
+using namespace NovelRT::Timing;
+
+class TimestampTest : public testing::Test {
+protected:
+  Timestamp _timeStamp;
+
+  void SetUp() override {
+    _timeStamp = Timestamp(TicksPerSecond);
+  }
+public:
+  TimestampTest::TimestampTest() : _timeStamp(Timestamp(0)) {}
```

Comment:
This is needed because `Timestamp` does not have a parameterless ctor.

Reply:
Why not add one which initializes the ticks to zero?

Reply:
Because, given the current API, it can't be reassigned a new tick value later. Do you want me to support this?